src/client/myself.tsx

```diff
@@ -76,16 +77,26 @@ export function MyselfProvider({ children }: { children: React.ReactNode }) {
   );
```

Comment:
The useMyId() function is currently also used inside useMyProps(). If I remove it, do I use useMyself() there, or directly get it from the server state as in the MyselfProvider? Does either impact the performance of the function?

Reply:
Or is `useContext(MyselfContext)` as it was previously in the `useMyself()` best here? With the addition of throwing an error when `player` is `null`?

Reply:
> The `useMyId()` function is currently also used inside `useMyProps()`.

Ah, I missed that.

> Or is `useContext(MyselfContext)` as it was previously in the `useMyself()` best here? With the addition of throwing an error when `player` is `null`?

I think that sounds good, if we also modify `MyselfContext` to no longer contain the entire player object but just the player id or `null`.

Reply:
Okay, I think I changed the `MyselfContext` so that is only contains the `playerId`. However, I seem to have broken prompts in the meantime? Could that be related to this somehow? (Prompt never show up, not for log in neither for creating dice templates from the roll interface)

Reply:
Ummm, I don't see how these could be related, weird. But `myself.tsx` looks good to me now 🎉

Reply:
Unfortunate :see_no_evil: